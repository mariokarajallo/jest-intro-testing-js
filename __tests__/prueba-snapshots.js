const cliente = {
  nombre: "Mario",
  balance: 1000,
  tipo: "Premium",
};

describe("Testing al cliente", () => {
  test("Es cliente es Mario", () => {
    expect(cliente).toMatchSnapshot();
  });
});
