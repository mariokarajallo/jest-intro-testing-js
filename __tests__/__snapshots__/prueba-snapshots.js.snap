// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es cliente es Mario 1`] = `
{
  "balance": 1000,
  "nombre": "Mario",
  "tipo": "Premium",
}
`;
